### PR TITLE
Chains 427 and 859

### PR DESCRIPTION
**Pull request by rampenke**

### CODE REVIEW

File: _data/chains/eip155-427.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Zeeth Chain",
+  "chain": "ZeethChain",
+  "icon": "zeethchain",
+  "rpc": [
+    "https://rpc.zeeth.io",
```
Comment (ligi) [inline _data/chains/eip155-427.json:6]:
```suggestion
    "https://rpc.zeeth.io"
```

File: _data/chains/eip155-859.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Zeeth Chain Dev",
+  "chain": "ZeethChainDev",
+  "icon": "zeethchaindev",
+  "rpc": [
+    "https://rpc.dev.zeeth.io",
```
Comment (ligi) [inline _data/chains/eip155-859.json:6]:
```suggestion
    "https://rpc.dev.zeeth.io"
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

also:
The Icon zeethchaindev does not exist - was used in eip155-859.json

**Review by rampenke — COMMENTED** [review]
Updated

**Comment by ligi** [comment]
The Icon zeethchaindev does not exist - was used in eip155-859.json

**Pull request closed without merging**